### PR TITLE
API-5190: Play 2.8 upgrade

### PR DESCRIPTION
**Pull request by mattclark-zerogravit**

### CODE REVIEW

File: it/uk/gov/hmrc/apipublisher/PublisherFeatureSpec.scala [hunk 1]
```diff
@@ -77,7 +77,7 @@ class PublisherFeatureSpec extends BaseFeatureSpec {
       )
 
       // TOOD - restore when api definition no longer rejects updated api
-      // Then("The api defintion is validated")
+      // Then("The api definition is validated")
```
Comment (AndySpaven) [inline it/uk/gov/hmrc/apipublisher/PublisherFeatureSpec.scala:80]: Remove this dead code ?

**Review by peteslater-ee — APPROVED** [review]

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,10 +1,10 @@
-resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.gov.uk/maven2"
+resolvers += "HMRC-open-artefacts-maven-2" at "https://open.artefacts.tax.service.gov.uk/maven2"
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
```
Comment (AndySpaven) [inline project/plugins.sbt:4]: Bump to 3.5 maybe ?

**Review by AndySpaven — COMMENTED** [review]
2 Suggested changes

File: it/uk/gov/hmrc/apipublisher/PublisherFeatureSpec.scala (hunk 1 above)
Comment (mattclark-zerogravit) [reply it/uk/gov/hmrc/apipublisher/PublisherFeatureSpec.scala]: Done

File: project/plugins.sbt (hunk 2 above)
Comment (mattclark-zerogravit) [reply project/plugins.sbt]: Done

**Pull request merged**